whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapConformanceEnum.java

```diff
@@ -0,0 +1,7 @@
+package net.ripe.db.whois.api.rdap;
+
+public enum RdapConformanceEnum {
+    CIRD_0("cidr0"), LEVEL_0("rdap_level_0"), FLAT_MODEL("nro_rdap_profile_asn_flat_0");
+
+    RdapConformanceEnum(final String value){}
```

Comment:
Can you store the value in a field? I think these are the expected values that should appear in the RDAP response?